app/services/analytics_events.rb

```diff
@@ -1306,9 +1306,111 @@ def idv_doc_auth_submitted_pii_validation(
+  # @param flow_path [String] "hybrid" for hybrid handoff, "standard" otherwise
+  # @param irs_reproofing [Boolean] Whether the user is being forced to reproof to access the IRS
+  # @param lexisnexis_instant_verify_workflow_ab_test_bucket [String] A/B test bucket for Lexis Nexis InstantVerify workflow testing
+  # @param opted_in_to_in_person_proofing [Boolean] Whether this user explicitly opted into in-person proofing
+  # @param proofing_results [Hash]
+  # @option proofing_results [String,nil] exception If an exception occurred during any phase of proofing its message is provided here
+  # @option proofing_results [Boolean] timed_out true if any vendor API calls timed out during proofing
+  # @option proofing_results [String] threatmetrix_review_status Result of Threatmetrix assessment, either "review", "reject", or "pass"
+  # @option proofing_results [Hash] context Full context of the proofing process
+  # @option proofing_results [String] context[:device_profiling_adjudication_reason] Reason code describing how we arrived at the device profiling result
+  # @option proofing_results [String] context[:resolution_adjudication_reason] Reason code describing how we arrived at the identity resolution result
+  # @option proofing_results [Boolean] context[:should_proof_state_id] Whether we need to verify the user's PII with AAMVA. False if the user is using a document from a non-AAMVA jurisdiction
+  # @option proofing_results [Hash] context[:stages] Object holding details about each stage of the proofing process
+  # @option proofing_results [Hash] context[:stages][:resolution] Object holding details about the call made to the identity resolution vendor
+  # @option proofing_results [Boolean] context[:stages][:resolution][:success] Whether identity resolution proofing was successful
+  # @option proofing_results [Hash] context[:stages][:resolution][:errors] Object describing errors encountered during identity resolution
+  # @option proofing_results [String,nil] context[:stages][:resolution][:exception] If an exception occured during identity resolution its message is provided here
+  # @option proofing_results [Boolean] context[:stages][:resolution][:timed_out] Whether the identity resolution API request timed out
+  # @option proofing_results [String] context[:stages][:resolution][:transaction_id] A unique id for the underlying vendor request
+  # @option proofing_results [Boolean] context[:stages][:resolution][:can_pass_with_additional_verification] Whether the PII could be verified if another vendor verified certain attributes
+  # @option proofing_results [Array<String>] context[:stages][:resolution][:attributes_requiring_additional_verification] Attributes that need to be verified by another vendor
+  # @option proofing_results [String] context[:stages][:resolution][:vendor_name] Vendor used (e.g. lexisnexis:instant_verify)
```

Comment:
I have never seen this syntax for nested keys, is it standard?

Reply:
Honestly, I'm not sure. I have found different examples showing different things. I think yard parses it more or less as an opaque string, so it comes through on the doc site _ok_. Is there a better way to do these kind of nested type definitions?

Reply:
I don't know of anything better, this is me just trying to understand. Do you have a screenshot of how it looks of the docs site?

Reply:
Right now it looks like this:

![image](https://github.com/18F/identity-idp/assets/364697/839afca6-2686-436d-a448-aef4afde6dcd)

[This issue](https://github.com/lsegal/yard/issues/622) suggests using bulleted lists, but I so far have not been able to get that to work

Reply:
![image](https://github.com/18F/identity-idp/assets/364697/e8b3a507-ee14-40a2-bbce-b83639e2b331)

Dots look a little cleaner to my eye, and would be copy/pasteable into cloudwatch

Reply:
the code to make the sample object is 100% ours so we could probably update that to custom handle things with dots in the middle too?